app/controllers/comments_controller.rb

```diff
@@ -247,4 +229,27 @@ def unhide
   def set_comment
     @comment = Comment.find(params[:id])
   end
+
+  def json_resource_created
+    { json: {
+      status: "created",
+      css: @comment.custom_css,
+      depth: @comment.depth,
+      url: @comment.path,
+      readable_publish_date: @comment.readable_publish_date,
+      published_timestamp: @comment.decorate.published_timestamp,
+      body_html: @comment.processed_html,
+      id: @comment.id,
+      id_code: @comment.id_code_generated,
+      newly_created: true,
+      user: {
+        id: current_user.id,
+        username: current_user.username,
+        name: current_user.name,
+        profile_pic: ProfileImage.new(current_user).get(width: 50),
+        twitter_username: current_user.twitter_username,
+        github_username: current_user.github_username
+      }
```

Comment:
WDUT about creating a function to return the user?

Reply:
Done!